### PR TITLE
Update alhambra board

### PR DESCRIPTION
**Pull request by imuguruza**
- Add "" to nextpnr option to avoid python3 parsing error
- Update PLL out value
- Update in Readme, as now the baudrate needs to be 115200 (tested in gtkterm and minicom)

### CODE REVIEW

File: servant.core [hunk 1]
```diff
@@ -147,7 +147,7 @@ targets:
     parameters : [memfile, memsize, PLL=ICE40_CORE]
     tools:
       icestorm:
-        nextpnr_options : [--hx8k, --package, tq144:4k, --freq, 16]
+        nextpnr_options : [--hx8k, --package, "tq144:4k", --freq, 16]
```
Comment (olofk) [inline servant.core:150]: nextpnr should be constrained to 32MHz since you changed the PLL frequency below.

Comment (imuguruza) [reply servant.core:150]: Makes sense

Comment (imuguruza) [reply servant.core]: I have added a commit to resolve it

**Comment by olofk** [comment]
All good. Thanks for this!

**Pull request merged**